posts/2020/09/design-of-the-versioned-hdf5-library.markdown

```diff
@@ -0,0 +1,175 @@
+<!--
+.. title: Design of the Versioned HDF5 Library
+.. slug: design-of-the-versioned-hdf5-library
+.. date: 2020-08-31 16:30:26 UTC-05:00
+.. tags:
+.. category:
+.. link:
+.. description:
+.. type: text
+
+-->
+
+In a previous post, we introduced the Versioned HDF5 library and described
+some of its features. In this post, we'll go into detail on how the underlying
+design of the library.
+
+<!-- TEASER_END -->
+
+The Versioned-HDF5 library is a library that wraps h5py and offers a versioned
+abstraction for HDF5 groups and datasets. The basic idea behind the design is
+that versioned-hdf5 is a
+[copy-on-write](https://en.wikipedia.org/wiki/Copy-on-write) system. The basic
+idea of copy-on-write is that all data is effectively immutable in the
```

Comment:
Probably should rephrase to not use `The basic idea` two sentences in a row for different concepts